openmdao/utils/options_dictionary.py

```diff
@@ -105,88 +105,61 @@ def __rst__(self):
         list of str
             A rendition of the options as an rST table.
         """
-        outputs = []
-        for option_name, option_data in sorted(self._dict.items()):
-            name = option_name
-            default = option_data['value'] if option_data['value'] is not _UNDEFINED \
-                else '**Required**'
-            values = option_data['values']
-            types = option_data['types']
-            desc = option_data['desc']
+        lines = self.to_table(fmt='rst').split('\n')
+        return lines
+
+    def to_table(self, fmt='github', missingval='N/A'):
+        """
+        Get a table representation of this OptionsDictionary as a table in the requested format.
 
+        Parameters
+        ----------
+        fmt : str
+            The formatting of the requested table.  Options are the same as those available
+            to the tabulate package.  See tabulate.tabulate_formats for a complete list.
+            Default value of 'github' produces a table in GitHub-flavored markdown.
+        missingval : str
+            The value to be displayed in place of None.
+
+        Returns
+        -------
+        str
+            A string representation of the table in the requested format.
+        """
+        try:
+            from tabulate import tabulate
+        except ImportError as e:
+            msg = "'to_table' requires the tabulate package but it is not currently installed." \
+                  " Use `pip install tablulate` or install openmdao with" \
+                  " `pip install openmdao[notebooks]`."
```

Comment:
`tabulate` should probably be added to the `docs` optional dependencies, since it is now required to build the docs